### DISCUSSION TITLE
Page reload while reading article

### DISCUSSION THREAD

**Discussion by andreascschmidt** [FreshRSS/FreshRSS#3386] [Q&A] [answered]
Before I open an issue I'd like to check if this behaviour is maybe "expected"

On my self-hosted Freshrss when reading articles I have at random times (I tried to time it) page refreshes (like page reloads) with at least Vivaldi and Brave browser (not sure if this is browser related)

So while I have for example a few articles read, currently inside an article (which marks it as read after a second) the whole page refreshes and all read articles are gone (which is expected by my settings) but this includes the article I was inside and might not have been finished reading.

Is there any way to stop that page refresh or reload, or is that a browser based reload setting?

**Comment by Frenzie** [comment]
I think it might be expected to refresh if you left the page alone for a long time but I'm sure some specifics don't add up.

**Comment by Alkarex** [comment]
Hum, the whole page is not supposed to refresh like that. It could be that for some reason your session expires, either due to server and/or browser settings. It can also be bad cache configuration. Some of that should be visible in your Web browser networking tools and/or Web server logs.
Which version of FreshRSS are you running, on which system (OS, Web server, Proxy...), and with which installation type (git, Docker...)?

**Reply by andreascschmidt** [reply]
I'm already playing with the config to test changes in the behavior. 
The Setup:
Version 1.17.1-dev
running on APACHE (no docker), Nginx as front, (hestiaCP) 
Behind Cloudflare (proxy turned off > dev mode).
According to the logs it looks like the browse initiates the page reload. (needs more digging into this)

I'm setting up a local instance to see what it does there...

**Comment by andreascschmidt** [comment, marked as answer]
I thought I close this as I have found my answer. 
"Never attribute to malice that which is adequately explained by stupidity"

I had the freshrss extension "autorefresh" running which was "totally surprisingly" refreshing .. 
https://github.com/Eisa01/FreshRSS---Auto-Refresh-Extension

Thanks anyway ..

**Reply by ischeriad** [reply]
This happened to me as well.

I went into the extension (.../xExtension-AutoRefresh/static/script.js) and changed the update interval from 1 minute (60000 milliseconds) to 10 minutes (600000 milliseconds).

It would be great to make this interval configurable from within FresRSS, but I haven't figured out how to do that.

**Reply by aledeg** [reply]
@ischeriad I've opened a PR on the repo to add the configuration of the refresh rate. See https://github.com/Eisa01/FreshRSS---Auto-Refresh-Extension/pull/4
